### PR TITLE
qml: change place of wallets loading in init

### PR DESCRIPTION
**Pull request by dvajdual**

### CODE REVIEW

File: GUI/QtQuick/qml/main.qml [hunk 1]
```diff
@@ -138,7 +138,7 @@ ApplicationWindow {
         running: false
         repeat: false
         onTriggered: {
-            bsApp.getArmoryServers()
+            //bsApp.getArmoryServers()
```
Comment (sergey-chernikov) [inline GUI/QtQuick/qml/main.qml:141]: What's the purpose of this change? When will it setup the default connection instead?

Comment (dvajdual) [reply GUI/QtQuick/qml/main.qml:141]: Bad commit...I have tried show wallets wizard without delay, but if I place function to other place also there is delay. So I dont like pr myself and I decline it)
